ru-ru/pyqt-ru.html.markdown

```diff
@@ -0,0 +1,83 @@
+---
+category: tool
+tool: PyQT
+filename: learnpyqt-ru.py
+contributors:
+    - ["Vadim Toptunov", "https://github.com/VadimToptunov"]
```

Comment:
can you please restore the contributors, and instead add yourself as a translator (`translators:`), like it's done https://raw.githubusercontent.com/adambard/learnxinyminutes-docs/master/ru-ru/bash-ru.html.markdown in the other files?